multiqc/modules/verifybamid/verifybamid.py

```diff
@@ -0,0 +1,238 @@
+            'description': 'Average sequencing depth at the sites in the VCF file',
+        }
+        # specify the number of decimal points to display
+        headers['FREEMIX']= {
+            'title': 'FREEMIX',
+            'description': 'Sequence-only estimate of contamination (0-1 scale)',
+            'format': '{:,.5f}' 
+        }
+        headers['FREELK1']= {
+            'title': 'FREEELK1',
+            'description': 'Maximum log-likelihood of the sequence reads given estimated contamination under sequence-only method',
+        }
+        headers['FREELK0']= {
+            'title': 'FREELK0',
+            'description': 'Log-likelihood of the sequence reads given no contamination under sequence-only method',
+        }
+        headers['FREE_RH']= {
+            'title': 'FREE_RH',
+            'description': 'Estimated reference bias parameter Pr(refBase|HET) (when --free-refBias or --free-full is used)',
+        }
+        headers['FREE_RA']= {
+            'title': 'FREE_RA',
+            'description': 'Estimated reference bias parameter Pr(refBase|HOMALT) (when --free-refBias or --free-full is used)',
+        }
+        # specify the number of decimal points to display
+        headers['CHIPMIX']= {
+            'title': 'CHIPMIX',
+            'description': 'Sequence+array estimate of contamination (NA if the external genotype is unavailable) (0-1 scale)',
+            'format': '{:,.5f}' 
+        }
+        headers[ 'CHIPLK1']= {
+            'title': 'CHIPLK1',
+            'description': 'Maximum log-likelihood of the sequence reads given estimated contamination under sequence+array method (NA if the external genotypes are unavailable)',
+        }
+        headers['CHIPLK0']= {
+            'title': 'CHIPLK0',
+            'description': ' Log-likelihood of the sequence reads given no contamination under sequence+array method (NA if the external genotypes are unavailable)',
+        }
+        headers['CHIP_RH']= {
+            'title': 'CHIP_RH',
+            'description': 'Estimated reference bias parameter Pr(refBase|HET) (when --chip-refBias or --chip-full is used)',
+        }
+        headers['CHIP_RA']= {
+            'title': 'CHIP_RA',
+            'description': 'Estimated reference bias parameter Pr(refBase|HOMALT) (when --chip-refBias or --chip-full is used)',
+        }
+        headers['DPREF']= {
+            'title': 'DPREF',
+            'description': 'Depth (Coverage) of HomRef site (based on the genotypes of (SELF_SM/BEST_SM), passing mapQ, baseQual, maxDepth thresholds.',
+        }
+        headers['RDPHET']= {
+            'title': 'RDPHET',
+            'description': 'DPHET/DPREF, Relative depth to HomRef site at Heterozygous site.',
+        }
+        headers['RDPALT'] = {
+            'title': 'RDPALT',
+            'description': 'DPHET/DPREF, Relative depth to HomRef site at HomAlt site.',
+        }
+
+        # if data was found
+        if len(data) > 0:
```

Comment:
No need for this check - we should have raised a `UserWarning` and exited before running this function if there was no data parsed.